content/company/strategy/enablement/index.md

```diff
@@ -54,6 +50,14 @@ When our CI pipeline breaks we need clear indications and rich observability of
 
 - FY22Q4 Key Result - Main branch is red for less than 1h per day for sourcegraph/sourcegraph.
 
+### Strengthen support of code management systems
+
+As we broaden our reach, prospects and customers look to us to support a large variety of source code management (SCM) systems, in a way that is robust and scalable. We want to provide our user base the means to simply, easily, and reliably connect to all common SCMs.
+
+### Improve the reliablity and performance of our code host connections and code storage
+
+Code host connectivity, scalability, and reliability underpin all other features within Sourcegraph. We need to ensure Sourcegraph can handle the size and quantity of repositories that future customers own and ensure we can meet any performance requirements customers have that impact code host connections and code storage.
+
```

Comment:
Can you incorporate the technical win in Q4 KR for one of our key prospects?

Reply:
Done